docs/intro/tutorials/gel_drizzle_booknotes.rst

```diff
@@ -0,0 +1,1264 @@
+
+2. Define the Gel schema
+========================
+
+
+Now that we have our project environment set up, let's define our database schema.
+For our Book Notes app, we'll create two main types:
+
+1. ``Book`` - to store information about books
+2. ``Note`` - to store notes associated with each book
+
+Let's edit the ``dbschema/default.esdl`` file that was created during initialization.
+
+.. edb:split-section::
+
+  Our schema defines two types:
+
+  - ``Book`` with properties like title, author, publication year, genre, and read date.
+  - ``Note`` with text content and a timestamp, linked to a specific book.
+
+  The relationship is defined such that a book can have multiple notes, and
+  each note belongs to exactly one book. We're using a computed link ``notes`` to
+  allow easy access to a book's notes.
+
+  .. code-block:: sdl
+      :caption: dbschema/default.esdl
+
+      module default {
+        type Book {
+          required title: str;
+          author: str;
+          year: int16;
+          genre: str;
+          read_date: datetime;
+
+          # Relationship to notes
+          multi notes := .<book[is Note];
+        }
+
+        type Note {
+          required text: str;
+          created_at: datetime {
+            default := datetime_current();
+          }
+
+          # Link to the book
+          required book: Book;
+        }
+      }
+
+.. edb:split-section::
+
+  Now let's apply this schema to our database by creating and applying a migration:
+
+  .. code-block:: bash
+
+      $ gel migration create
+      $ gel migrate
+
+
+3. Install and set up Drizzle
+=============================
+
+Now that we have our Gel schema in place, we can integrate Drizzle ORM with our
+Next.js application. Drizzle will provide a type-safe way to interact with our
+Gel database.
+
+.. edb:split-section::
+
+  First, let's install Drizzle and its dependencies. We'll need ``drizzle-orm`` and
+  ``drizzle-kit`` for this.
+  Drizzle ORM is the core library, while Drizzle Kit is a CLI tool that helps
+  with schema generation and migrations.
+
+  .. code-block:: bash
+
+      $ npm i drizzle-orm drizzle-kit
+
+.. edb:split-section::
+
+  Let's create a Drizzle configuration file in the root of our project. We'll set the ``dialect`` to ``gel``
+  to tell Drizzle that we're using Gel as our database.
+
+  .. code-block:: typescript
+      :caption: drizzle.config.ts
+
+      import { defineConfig } from 'drizzle-kit';
+
+      export default defineConfig({
+        dialect: 'gel',
+      });
+
+.. edb:split-section::
+
+  Now, let's pull the database schema into Drizzle. This step will introspect
+  our Gel database and generate TypeScript files that we can use with Drizzle.
+
+  .. code-block:: bash
+
+      $ npx drizzle-kit pull
+
+.. edb:split-section::
+
+  The schema file that Drizzle generated will look something like this. We've added
+  relations to make it easier to work with related data between books and notes.
```

Comment:
> We've added relations to make it easier to work with related data between books and notes.

This sentence is confusing. We've added them to the .gel file, not the drizzle reflected schema, right?

Moreover, we should put a note here emphasizing that users are not supposed to modify the reflected schema: the flow is always [change .gel file] -> [run drizzle kit]